appengine/flexible/Analytics/runTests.ps1

```diff
@@ -14,10 +14,10 @@
 
 Import-Module -DisableNameChecking ..\..\..\BuildTools.psm1
 
-dotnet restore
+dotnet restore --force
```

Comment:
Can't say I've ever seen this make a difference, but I wouldn't expect it to cause any problems, at least.

Reply:
Yep, I'm not sure either tbh, but it does says on docs that the other commands (build, test, run...) perform a restore "if necessary". We have #1278 that might be due to restore not being done properly. If this doesn't help I'll try forcing it to always fetch packages from Nuget.